docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_privatelink/_index.md

```diff
@@ -0,0 +1,245 @@
+    ```
+
+    For example:
+
+    ```shell
+    az ad sp create-for-rbac -n "http://AzureArcServers" --role contributor
+    ```
+
+    Output should look like this:
+
+    ```json
+    {
+    "appId": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "displayName": "AzureArcServers",
+    "name": "http://AzureArcServers",
+    "password": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "tenant": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+    }
+    ```
+
+- Azure Arc-enabled servers depends on the following Azure resource providers in your subscription in order to use this service. Registration is an asynchronous process, and registration may take approximately 10 minutes.
+
+  - Microsoft.HybridCompute
+  - Microsoft.GuestConfiguration
+
+      ```shell
+      az provider register --namespace 'Microsoft.HybridCompute'
+      az provider register --namespace 'Microsoft.GuestConfiguration'
+      ```
+
+      You can monitor the registration process with the following commands:
+
+      ```shell
+      az provider show --namespace 'Microsoft.HybridCompute'
+      az provider show --namespace 'Microsoft.GuestConfiguration'
+      ```
+
+  > **Note: It is optional but highly recommended to scope the SP to a specific [Azure subscription and resource group](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest).**
+
+## Automation Flow
+
+For you to get familiar with the automation and deployment flow, below is an explanation.
+
+1. User is editing the ARM templates parameters file (1-time edit). These parameters values are being used throughout the deployment. Two separate files are provided, one per resource group.
+
+2. User creates two resource groups in two different regions:
+    - Azure Resource Group
+    - On-premises Resource Group
+
+3. User deploys the ARM template for the resources in the Azure Resource Group. The ARM template will create:
+
+    - Azure SQL server and a SQL database
+    - Azure Private Link
+    - Azure Private DNS zone
+    - Azure VPN Gateway and its public IP address
+    - Azure VNET
+
+4. User deploys the ARM template for the resources in the On-premises resource group. The ARM template will create:
+
+    - Azure VNET
+    - Azure Bastion
+    - Azure VPN Gateway and its public IP address
+    - Azure Linux Virtual Machine with a custom script extension that will run the [_install_arc_agent.sh_](https://github.com/microsoft/azure_arc/blob/main/azure_arc_servers_jumpstart/azure/linux/arm_template/scripts/install_arc_agent.sh) shell script to Arc-enable the Azure VM.
+
+        > **Note: The [_install_arc_agent.sh_](https://github.com/microsoft/azure_arc/blob/main/azure_arc_servers_jumpstart/azure/linux/arm_template/scripts/install_arc_agent.sh) shell script will enable the OS firewall and set up new rules for incoming and outgoing connections. By default all incoming and outgoing traffic will be allowed, except blocking Azure IMDS outbound traffic to the *169.254.169.254* remote address.**
```

Comment:
i would add the Azure VM disclaimer we have in the Azure VMs scenarios

Reply:
added